cmd/scip/print_test.go

```diff
@@ -47,10 +47,10 @@ func TestJSONPrinting(t *testing.T) {
 
 	type JsonIndex struct {
 		Metadata struct {
-			ProjectRoot string `json:"projectRoot"`
+			ProjectRoot string `json:"project_root"`
```

Comment:
This is technically a breaking change, but `scip print --json` is only meant to be used for debugging, so I think this is OK. This is coming about because in `scip.pb.go`, the struct tags specified for `protobuf`'s JSON encoding and the normal JSON encoding are different.

https://sourcegraph.com/github.com/sourcegraph/scip/-/blob/bindings/go/scip/scip.pb.go?L1488

```
	ProjectRoot string `protobuf:"bytes,3,opt,name=project_root,json=projectRoot,proto3" json:"project_root,omitempty"`
```